scripts/releases/prebuild/compose-framework.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ * @oncall react_native
+ */
+
+const {execSync} = require('child_process');
+const fs = require('fs');
+const path = require('path');
+
+/*::
+import type { Dependency, Platform } from './types';
+*/
+
+/**
+ * Composes the final XCFramework from the artifacts in the build folder
+ */
+async function createFramework(
+  scheme /*: string */,
+  dependencies /*: $ReadOnlyArray<Dependency> */,
+  rootFolder /*: string */,
+  buildFolder /*: string */,
+) {
+  console.log('✅ Composing iOS XCFramework...');
+
+  // Get the build destination path from the prebuild step
+  const frameworksOutputFolder = path.join(buildFolder, 'Build', 'Products');
+  const frameworks = fs.readdirSync(frameworksOutputFolder);
+  console.log('Frameworks found:', frameworks.join(', '));
+
+  const frameworkPaths = frameworks.map(framework =>
+    path.join(frameworksOutputFolder, framework),
+  );
+
+  const output = path.join(rootFolder, `${scheme}.xcframework`);
+
+  // Check if output already exists and delete it
+  fs.rmSync(output, {recursive: true, force: true});
+
+  console.log('Output path:', output);
+
+  const frameworksArgs = frameworkPaths
+    .map(
+      framework =>
+        `-framework ${path.join(
+          framework,
+          'PackageFrameworks',
+          `${scheme}.framework`,
+        )} -debug-symbols ${path.join(framework, `${scheme}.framework.dSYM`)}`,
```

Comment:
I would not add the debug symbols in the frameworks.
They will grow the download size dramatically and they are not really needed. We will publish the debug symbols separatedly so users can download them and use them to manually symbolicate their crashes (sentry let's you upload the symbols and then it symbolicate the crashes for you).
We need to keep the binary size small, if we can.

Reply:
Symbols are now exported to the ./Symbols folder